### ISSUE TITLE
Link on main page not pointing to localized version which is available

### ISSUE THREAD

**Issue by DaddyAG** [thunderbird/thunderbird-website#198]
Visit https://www.thunderbird.net/hr/
Bellow main download button "BESPLATNO PREUZIMANJE" you will see link named "Sustavi i jezici" ("Systems & Languages") which points to https://www.thunderbird.net/en-US/thunderbird/all/

HR (Croatian) Thunderbird main page points to en-US "System & Languages" subpage, which localized version is available at https://www.thunderbird.net/hr/thunderbird/all/

If you just try to visit https://www.thunderbird.net/thunderbird/all/ which doesn't have locale entered, as HR user you will be automaticly redirected to https://www.thunderbird.net/hr/thunderbird/all/ so it seams to me that en-US is hardcoded for some reason.

If I visit other locales, like DE, Systems & Languages is still hardcoded to en-US.

**Comment by wsmwk** [comment]
@MelissaAutumn, this is invalid with the completely new website?

**Comment by MelissaAutumn** [comment]
Yep, looks like it was also fixed waaay back when https://github.com/thunderbird/thunderbird-website/commit/885e4485ee84f1c0584db0aba308ec879f2f8883
